### ISSUE TITLE
Cookie Writer: Allow customization of expiration date

### ISSUE THREAD

**Issue by zhouyx** [ampproject/amphtml#22422]
The `cookeWriter` feature allow publisher to write cookie on the origin. 

Right now, the expiration date is by default set to 1 year. We need to provide a way for publisher's to customize the cookie expiration date. 

Proposed configuration
```
  'cookies': {
    'cookieMaxAge': 365*24*3600,
    'keyValue1': {
      'value': 'QEURY_PARAM(fake1)',
    }
  },
```
`cookieMaxAge` accepts a number unit in second. Which is the cookie will be expired after that amount of time. 

Put `cookieMaxAge: 604800` (604800=60*60*24*7) if one wants the cooke to expire after 7 days. 

cc @zikas

**Comment by zhouyx** [comment]
Closed by #22483 
@zikas FYI: change in prod.

**Comment by AlexBenny** [comment]
Thanks for adding this feature. I'm a bit in doubt about how it should be used.

I would write a configuration like these one making the max age configurable:
```
"cookies": {
    "enabled": true,
    "cookieMaxAge": {
      "value": "$DEFAULT(${cookieMaxAge}, $CALC(86400, 365, multiply, true))"
    },
   ...
  },
```
reading the implementation of the `cookie-writer.js` I see that the value is directly converted to a number and the variables are not expanded:
```
const cookieMaxAgeNumber = Number(inputConfig['cookieMaxAge']);
```

Would it be possible to set the `cookieMaxAge` making that custom for the user?

I'm new on this technology so, please, let me know if there is something obvious I haven't considered or if I should send this question in a different channel or GH issue.
